scale_build/clean.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import os
+import shutil
+
+from .utils.paths import HASH_DIR, LOG_DIR, PKG_DIR, SOURCES_DIR, TMP_DIR
+
+
+logger = logging.getLogger(__name__)
+
+
+def clean_bootstrap_logs():
+    for f in filter(lambda f: f.startswith('bootstrap'), os.listdir(LOG_DIR)):
+        os.unlink(os.path.join(LOG_DIR, f))
+
+
+def clean_packages():
+    for d in (HASH_DIR, PKG_DIR):
+        shutil.rmtree(d, ignore_errors=True)
```

Comment:
What errors are you ignoring?

We expect `HASH_DIR` and `PKG_DIR` to be non-existent after this method terminates successfully. Will it always be so?